packages/flutter_tools/lib/src/xcode_project.dart

```diff
@@ -288,7 +311,7 @@ class IosProject extends XcodeBasedProject {
     }
 
     if (scheme == null) {
-      scheme = info.schemeFor(buildInfo);
+      scheme = info.schemeFor(flavor: buildInfo?.flavor, hostAppProjectName: FlutterProject.current().ios.hostAppProjectName);
```

Comment:
Isn't hostAppProjectName a property on this instance?
```suggestion
      scheme = info.schemeFor(flavor: buildInfo?.flavor, hostAppProjectName: hostAppProjectName);
```